### PR TITLE
Incorporate CHAOS Eng team work - Phase 1

### PR DESCRIPTION
**Pull request by ferantivero**
closes: #344940

We're please to incorporate the CHAOS Eng team work on top of this Microservices Reference Implementation. In this very initial phase, we'd like to ingest the following contributions while leaving out some for now:

* format changes to keep upstream easy to contrib to
* incorporate inner loop scripts leaving out Docker approach for now
* update the resources requests/limit for workflow in dev
* get rid of the assets altering the main narrative
* remove upstream manifests from our repo
* general updates/bug fixes on top of the k8s manifest(s)

### CODE REVIEW

File: azuredeploy-prereqs.json [hunk 1]
```diff
@@ -1,166 +1,165 @@
...
+          "outputs": {
+            "deliveryIdName": {
+              "value": "[variables('environmentSettings')[parameters('environmentName')].deliveryIdName]",
+              "type": "string"
+            },
+            "deliveryPrincipalResourceId": {
+              "value": "[concat(subscription().id, '/resourceGroups/',parameters('resourceGroupName'),'/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',variables('environmentSettings')[parameters('environmentName')].deliveryIdName)]",
+              "type": "string"
+            },
+            "websiteIdName": {
```
Comment (ckittel) [inline azuredeploy-prereqs.json:130]: Will this return anything at this point?

**Review by ckittel — APPROVED** [review]
Just one question, but shouldn't block deployment.

**Comment by ckittel** [comment]
If you still wanted to walk through this, just lmk.

File: azuredeploy-prereqs.json (hunk 1 above)
Comment (ferantivero) [reply azuredeploy-prereqs.json]: thanks for the feedback and the laser focus @ckittel,

it is now addressed from [b2f8d](https://github.com/mspnp/microservices-reference-implementation/pull/191/commits/b2f8dddef532b29449ed9b048fc6c1bd98b36276)

**Pull request merged**